src/custom-types.json

```diff
@@ -493,6 +510,13 @@
   "audio/ogg": {
     "compressible": false
   },
+  "audio/rtx": {
+    "extensions": ["rtx"],
```

Comment:
Neither of the sources for this entry seem to list a file extension of `.rtx` anywhere. I may be missing it though, as the docs are long. Please point out where the file extension is listed for this or add an additional primary source with the link between the file extension and the mime type.